datacite-rest-client/src/it/java/org/gbif/datacite/rest/client/retrofit/DataCiteRetrofitClientSteps.java

```diff
@@ -57,24 +62,32 @@ public void performGetDoiList() {
     }
 
     @When("^Perform a request to DataCite's POST DOI$")
-    public void performCreateDoi() {
-        Datacite42Schema datacite42Schema = new Datacite42Schema();
-        datacite42Schema.setId(DOI_105);
-        datacite42Schema.setDoi(DOI_105);
+    public void performCreateDoi() throws IOException {
+        DoiSimplifiedModel model = new DoiSimplifiedModel();
+        model.setDoi(DOI_105);
+        model.setId(DOI_105);
+        String xmlMetadata = IOUtils.toString(
+                this.getClass().getResourceAsStream("/datacite-example-full-v4.xml"),
+                "UTF-8");
```

Comment:
there's constant for UTF-8 in java.nio.charset.StandardCharsets

Reply:
done